FLASK/main.py

```diff
@@ -0,0 +1,304 @@
+from flask import Flask
```

Comment:
You may Split  this class into several classes -  authorization, registration, attractions etc.